dotnet/samples/KernelSyntaxExamples/Example70_Assistant.cs

```diff
@@ -120,13 +120,10 @@ private static async Task RunAsFunctionAsync()
 
         // Create assistant, same as the other cases.
         var assistant =
-            await AssistantBuilder.FromDefinitionAsync(
-                TestConfiguration.OpenAI.ApiKey,
-                OpenAIFunctionEnabledModel, // Must be a function-enabled model.
-                EmbeddedResource.Read("Assistants.ParrotAssistant.yaml")); // Defined under ./Resources/Assistants
-
-        // Import assistant as a KernelFunction
-        var plugin = KernelPluginFactory.CreateFromObject(assistant, assistant.Id);
+            await new AssistantBuilder()
+                .AddOpenAIChatCompletion(OpenAIFunctionEnabledModel, TestConfiguration.OpenAI.ApiKey)
```

Comment:
Should the builder method name follow the same "With*" pattern as the other methods in `AssistantBuilder` and `KernelBuilder`?

Reply:
I was wondering the same.  Matthew picked the name, so I've just been rolling with it...let's take that alignment now. Yes.

Reply:
Done